css/at-rules/font-face.json

```
@@ -911,6 +911,54 @@
                 "deprecated": false
               }
             }
+          },
+          "format_variations": {
+            "__compat": {
+              "description": "<code>format('*-variations')</code>",
+              "support": {
+                "chrome": {
+                  "version_added": "66"
+                },
+                "chrome_android": {
+                  "version_added": "66"
+                },
+                "edge": {
+                  "version_added": "17"
+                },
+                "firefox": {
+                  "version_added": "62"
+                },
+                "firefox_android": {
+                  "version_added": "62"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": "53"
+                },
+                "opera_android": {
+                  "version_added": "53"
+                },
+                "safari": {
+                  "version_added": "11"
+                },
+                "safari_ios": {
+                  "version_added": "11"
+                },
+                "samsunginternet_android": {
+                  "version_added": "9.0"
+                },
+                "webview_android": {
+                  "version_added": "66"
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
```

One question for you, @danburzo: is this specified somewhere? The spec linked to from the parent feature doesn't list this string format. Is there another spec we ought to add for this feature?

https://drafts.csswg.org/css-fonts/#font-format-definitions

---

Yes, it was specified in the previous revision of css-fonts-4, linked below. Given the fact that it was in an outdated spec, I was not sure how place the feature status flags.

https://www.w3.org/TR/2021/WD-css-fonts-4-20210729/#font-face-src-parsing

---

Sorry wrong link, I meant the April 2018 Working Draft: https://www.w3.org/TR/2018/WD-css-fonts-4-20180410/#src-desc

---

Thanks for looking into this, @danburzo. Digging into that spec's history, it looks like it was specified but then removed for reasons which I haven't had time to learn. See https://github.com/w3c/csswg-drafts/issues/6328#issuecomment-856037966. I think we ought to label this as `"standard_track": false` (until such time it returns).

```suggestion
                "standard_track": false,
```

---

Sounds good. I haven't had the chance either to source the discussions that led to the changes in syntax.